azure-pipelines.bundlesize.yml

```diff
@@ -1,5 +1,6 @@
 pr:
   - master
```

Comment:
can we add comment or link to an issue so this will be removed once merged to master ?